src/Annotations/Schema.php

```diff
@@ -483,6 +490,18 @@ public function validate(array $stack = [], array $skip = [], string $ref = '',
             return false;
         }
 
+        if ($this->nullable !== Generator::UNDEFINED && $this->_context->isVersion(OpenApi::VERSION_3_1_0)) {
+            $this->_context->logger->error($this->identity() . ' must not have the "nullable" property when using OpenApi version ' . OpenApi::VERSION_3_1_0 . ' in ' . $this->_context);
+
+            return false;
+        }
+
+        if (is_array($this->type) && $this->_context->isVersion(OpenApi::VERSION_3_0_0)) {
+            $this->_context->logger->error($this->identity() . ' "type" must be of type string in OpenApi version ' . OpenApi::VERSION_3_0_0 . ', array given in ' . $this->_context);
+
+            return false;
+        }
```

Comment:
@DerManoMann I could also get rid of this validation step for now and then simply leave this PR as is.

This would atleast allow users of this package to manually define their types according to openapi 3.1.0 standard & it would fix the `TypeError` issue https://github.com/nelmio/NelmioApiDocBundle/issues/2152, thoughts?

Reply:
I have decided to remove this validation step to prevent breaking changes